### PR TITLE
Psycopg2 scripts-Flask_scripts PR fixes.

### PR DESCRIPTION
**Pull request by kapkic**
Let's review this with flask_scripts2. This fixes some issues on flask_scripts as well. We can merge this onto flask_scripts2, then merge it with develop.
copy_all.sh is especially useful as a lindfs cp #203 issue workaround.

### CODE REVIEW

File: src/scripts/copy_all.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+lindfs cp /home/lind/lind_project/tests/applications/python/Lib /usr/local/lib/python2.7
+lindfs cp /usr/lib/python2.7/config/ /usr/lib/python2.7/config/
+lindfs cp /usr/include/python2.7/ /usr/include/python2.7/
+lindfs cp /home/lind/lind_project/src/scripts/includes/passwd /etc/passwd
+
+#Python Libs
+echo Copying files...
+libsarray=('_collections' 'operator' 'itertools' '_struct' 'math' 'binascii' 'time' 'cStringIO' '_random' 'array' '_socket' '_functools' 'cPickle' '_struct' 'select' 'array' 'unicodedata')
+length=${#libsarray[@]}
+for (( i = 0; i < length; i++ ));
```
Comment (jesings) [inline src/scripts/copy_all.sh:20]: you should just do for var in libsarray and use $var within the loop

File: src/scripts/load_python_libs.sh [hunk 2]
```diff
@@ -3,3 +3,19 @@
 ./load_openssl.sh
 cd /home/lind/lind_project/tests/applications/python
 ./fastcompile.sh
+
+# Copies 
+echo Copying files...
+libsarray=('_collections' 'operator' 'itertools' '_struct' 'math' 'binascii' 'time' 'cStringIO' '_random' 'array' '_socket' '_functools' 'cPickle' '_struct' 'select' 'array' 'unicodedata')
+length=${#libsarray[@]}
+for (( i = 0; i < length; i++ ));
```
Comment (jesings) [inline src/scripts/load_python_libs.sh:11]: Again you should iterate over elements not indices

File: tests/applications/python-modules/psycopg2/bootstrap_nacl [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+echo Compiling...
+length=${#modulesarray[@]}
+for (( i = 0; i < length; i++ ));
+do
+        x86_64-nacl-gcc -pthread -DPY_FORMAT_LONG_LONG=ll -fno-strict-aliasing -D_FORTIFY_SOURCE=2 -g -Wformat -Werror=format-security -fPIC -DPSYCOPG_DEFAULT_PYDATETIME=1 -DPG_VERSION_NUM=120009 -DHAVE_LO64=1 -I/home/lind/lind_project/tests/applications/python/Include -I/home/lind/lind_project/tests/applications/python -I. -I/home/lind/lind_project/tests/applications/postgres -I/home/lind/lind_project/tests/applications/postgres/src/include -I/home/lind/lind_project/tests/applications/postgres/src/interfaces/libpq -c psycopg/${modulesarray[i]}.c -o build/${modulesarray[i]}.o -Wdeclaration-after-statement
+done
```
Comment (jesings) [inline tests/applications/python-modules/psycopg2/bootstrap_nacl:12]: Same business with indices and values

File: tests/applications/python/fastcompile.sh [hunk 4]
```diff
@@ -18,55 +18,29 @@ do
...
- x86_64-nacl-gcc -pthread -shared -Wl,-O1 -Wl,-Bsymbolic-functions -std=c99 build/zlibmodule.o -L. -LModules/zlib -lpython2.7 -lz -o build/lib/zlib.so
- lindfs cp /home/lind/lind_project/tests/applications/python/build/lib/zlib.so /usr/local/lib/python2.7/lib-dynload/zlib.so
- lindfs cp /home/lind/lind_project/tests/applications/python/Modules/zlib/libz.so.1.2.3 /lib/glibc/libz.so.1
+ioarray=('bufferedio' 'bytesio' 'fileio' 'iobase' '_iomodule' 'stringio' 'textio')
+length=${#ioarray[@]}
+for (( i = 0; i < length; i++ ));
+do
+    x86_64-nacl-gcc -DPY_FORMAT_LONG_LONG=ll -std=c99 -fPIC -fno-strict-aliasing -march=x86-64 -mtune=generic -O2 -pipe -DNDEBUG -I. -IInclude -I./Include -IModules/_io -c Modules/_io/${ioarray[i]}.c -o build/${ioarray[i]}.o
+done
+x86_64-nacl-gcc -pthread -shared -Wl,-O1 -Wl,-Bsymbolic-functions -std=c99 build/bufferedio.o build/bytesio.o build/fileio.o build/iobase.o build/_iomodule.o build/stringio.o build/textio.o -L. -lpython2.7 -o build/lib/_io.so
```
Comment (jesings) [inline tests/applications/python/fastcompile.sh:33]: same deal with values instead of indices

**Review by jesings — COMMENTED** [review]
Awesome stuff!

File: src/scripts/copy_all.sh [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+#Bash
+lindfs cp /home/lind/lind_project/tests/applications/bash/bash /bin/bash
+
+#Python
+lindfs cp /home/lind/lind_project/tests/applications/python/python /bin/python;
+lindfs cp /home/lind/lind_project/tests/applications/python/libpython2.7.so.1.0 /lib/glibc/libpython2.7.so.1.0
+lindfs cp /home/lind/lind_project/tests/applications/python/Lib /usr/local/lib/python2.7
+lindfs cp /usr/lib/python2.7/config/ /usr/lib/python2.7/config/
+lindfs cp /usr/include/python2.7/ /usr/include/python2.7/
```
[the remark this answers is not in the document]
Comment (kapkic) [reply src/scripts/copy_all.sh:13]: I will look into this, I've added these when building python for the first time-but we're sure that some files inside the folders are required by python.

File: tests/applications/python/fastcompile.sh (hunk 4 above)
Comment (kapkic) [reply tests/applications/python/fastcompile.sh]: Done.

File: tests/applications/python-modules/psycopg2/bootstrap_nacl (hunk 3 above)
Comment (kapkic) [reply tests/applications/python-modules/psycopg2/bootstrap_nacl]: Done

File: src/scripts/load_python_libs.sh (hunk 2 above)
Comment (kapkic) [reply src/scripts/load_python_libs.sh]: Done.

File: src/scripts/copy_all.sh (hunk 1 above)
Comment (kapkic) [reply src/scripts/copy_all.sh]: Done.

File: src/scripts/copy_all.sh (hunk 5 above)
[the remark this answers is not in the document]
Comment (kapkic) [reply src/scripts/copy_all.sh]: Okay, it turns out we're copying unnecessary and _wrong_ files. Thanks for pointing it out!

**Review by jesings — APPROVED** [review]
Looks great now!
I'll try to debug lind_fs_utils soon

**Review by rennergade — APPROVED** [review]
Cool. These changes look good from what we talked about!

**Pull request merged**